technical-reference/reference-technical-articles/automatic-income-allocation.md

```diff
@@ -0,0 +1,87 @@
+# Automatic income allocation
+
+The purpose of this document is to define the technical solution for distributing the incomes generated by the Venus protocol. The distribution takes into account the incomes generated by the spread (borrow minus supply rates) and liquidations (part of the liquidation incentives). The document covers the distribution process for both Core pools and Isolated pools, and explains how the incomes are allocated to different destinations based on specific rules and percentages. Additionally, it outlines the approach for near-streaming income distribution.
+
+## Income Distribution Schema
+We want to distribute the protocol incomes this way *[Tokenomics](https://snapshot.org/#/venus-xvs.eth/proposal/0xc9d270ccecb7b91c75b95b8d9af24fc7c20cd38c0c0c44888ed4e7724f4e7ce9)*
```

Comment:
```suggestion
```

Reply:
Maybe we can add a link to the tokenomics page: https://docs-v4.venus.io/governance/tokenomics

And remove the percentage and details from here, to avoid duplicated info (it can be more difficult to maintain in the future)